e2e/cypress/support/api/user.js

```diff
@@ -253,12 +253,11 @@ Cypress.Commands.add('apiCreateUser', ({
 
 Cypress.Commands.add('apiCreateGuestUser', ({
     prefix = 'guest',
+    bypassTutorial = true,
     hideCloudOnboarding = true,
-    activate = true,
 } = {}) => {
-    return cy.apiCreateUser({prefix, hideCloudOnboarding}).then(({user}) => {
+    return cy.apiCreateUser({prefix, bypassTutorial, hideCloudOnboarding}).then(({user}) => {
         cy.apiDemoteUserToGuest(user.id);
-        cy.externalActivateUser(user.id, activate);
```

Comment:
This is causing issue in Cloud server. Removing since this is actually not being used.